src/ui/src/components/chart/SankeyChart.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import {
+  JinjatECharts,
+  JinjatEChartsProps,
+} from "@components/chart/JinjatECharts";
+
+interface JinjatSankeyChartsProps extends JinjatEChartsProps {
+  data: undefined;
+  sourceCol: "source";
+  targetCol: "target";
+  valueCol: "value";
+}
+
+export const JinjatSankeyChart: React.FC<JinjatSankeyChartsProps> = (props) => {
+  const options = props.options || {};
+  if (options.series != null) {
+    return (
+      <div>
+        `options.series` is not null, please use `ECharts` component instead.
+      </div>
+    );
+  }
+  options.series = [
+    {
+      type: "sankey",
+      encode: {
+        data: props.data,
+        sourceCol: props.sourceCol,
```

Comment:
Same with funnel, does EChart support these configs?

Reply:
I need clarification on this. I see these properties in the Evidence example, but as you said, not in the Echarts config. I am unsure about how can I find the actual property from Echarts API docs that match with the Evidence example.